modules/avatar/avatar.go

```diff
@@ -11,6 +11,7 @@ import (
 	"image/color/palette"
 
 	// Enable PNG support:
+	_ "image/jpeg"
```

Comment:
unrelated fix/enhancement !?!?

Reply:
No. It's related. Once `setting` don't depend on `git`, no `image/jpeg` imported to avatar and then read jpeg file will fail. I guess because `git` has already imported the package.

Reply:
I have something similiar [here](https://github.com/go-gitea/gitea/pull/14867/commits/d300009041c1c87d9331217d9fa34c9fa015ef0d)